Dockerfile.arm64

```diff
@@ -0,0 +1,78 @@
+ARG PY_VERSION_DEFAULT=3.9
+
+FROM dokken/centos-stream-9 as base
+
+USER root
+
+ARG PY_VERSION_DEFAULT
+ARG MEMGRAPH_RPM
+ENV MG_VERSION ${MG_VERSION}
+ENV PY_VERSION ${PY_VERSION_DEFAULT}
+
+COPY ${MEMGRAPH_RPM} /
+
+#essentials for production/dev
+RUN yum update && yum install -y \
+    libcurl        `memgraph` \
+    openssl         `memgraph` \
+    make `mage-memgraph` \
+    cmake           `mage-memgraph` \
+    curl            `mage-memgraph` \
+    g++             `mage-memgraph` \
+    python3         `mage-memgraph` \
+    python3-pip     `mage-memgraph` \
+    python3-setuptools     `mage-memgraph` \
+    python3-devel     `mage-memgraph` \
```

Comment:
Where is the guarantee that the Python version matches the above-mentioned PY_VERSION_DEFAULT? 

@antoniofilipovic